app/src/main/res/layout/act_calendar_week.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
yeah changing the name was a good idea, I was thinking of it too bc it was starting to annoy me, so thx for doing it :)

Reply:
haha, no probs :)